resources/js/Pages/Guest/ResetPassword.vue

```diff
@@ -23,14 +32,24 @@ function submit() {
 
 <template>
   <Head>
```

Comment:
Maybe create some wrapper component for that.

Reply:
I decided to get rid of meta tags in Head, so no need in extra component